lib/improver/tests/blending/weighted_blend/test_unify_forecast_reference_time.py

```diff
@@ -1,135 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
As this utility has moved I've copied these unit tests to lib/improver/tests/utilities/test_temporal.py